dotcom-rendering/src/amp/lib/real-time-config.ts

```diff
@@ -0,0 +1,143 @@
+import { AdRegion } from './region-classes';
+
+/**
+ * Type of AMP ad
+ *
+ * These values determine the computed RTC parameters
+ */
+type AdType =
+	| { isSticky: true }
+	| {
+			isSticky?: false;
+			adRegion: AdRegion;
+	  };
+
+/**
+ * The properties required by different Prebid URLS / vendors
+ *
+ * These can be computed from the Config type above
+ */
+export type RTCParameters = {
+	placementId: number;
+	tagId: string;
+	profileId: string;
+	pubId: string;
+};
+
+/**
+ * Determine the Xandr Placement ID that is used to look up a given stored bid request
+ *
+ * Stored bid requests are stored by the prebid server instance and each is
+ * keyed by a placement ID. This placement ID corresponds to the tag id parameter
+ * provided on the client
+ *
+ */
+const getPlacementId = (adType: AdType): number => {
+	if (adType.isSticky) {
+		return 9;
+	}
+	switch (adType.adRegion) {
+		case 'US':
+			return 7;
+		case 'AU':
+			return 6;
+		default:
+			return 4;
+	}
+};
+
+/**
+ * Determine the Relevant Yield tag id that is used to look up a given stored bid request
+ */
+const getTagId = (adType: AdType): string => {
+	if (!adType.isSticky) {
+		switch (adType.adRegion) {
+			case 'UK':
+				return '6214ca675cf18e70cbaeef37_6214c9a4b73a6613d4aeef2f';
+			case 'US':
+				return '6214cb381a577cd525aeef3f_6214caacb52b565527aeef39';
+			case 'AU':
+				return '6214cbe6a24103508faeef45_6214cb50aac9c1160daeef40';
+			// 'INT' same as sticky
+			default:
+				break;
+		}
+	}
+	return '6214ca56243f4ff4f5aeef36_6214c723c70856442e4d79f2';
+};
```

Comment:
Good point, Have updated the PR with changes to address both your comments!

I think the `switch` is clear in `getPlacementId` because a sticky ad never generates the same id as a regional ad. I think in this case and the other you've flagged it'd be simpler if I got rid of the `switch` altogether and stuck to an `if`.